server/run.sh

```
@@ -1,4 +1,12 @@
 source .venv/bin/activate
+# needed for autonomy setup
+if [ -f "/opt/ros/humble/setup.bash" ]; then
+    source /opt/ros/humble/setup.bash
+    python3 ./py_server.py "$@" --autonomy
+else
+    echo "\033[91mNo autonomy source file found at /opt/ros/humble/setup.bash"
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)